core/domain/exp_services.py

```diff
@@ -435,6 +572,12 @@ def export_states_to_yaml(exploration_id, version=None, width=80):
     """Returns a python dictionary of the exploration, whose keys are state
     names and values are yaml strings representing the state contents with
     lines wrapped at 'width' characters.
+
+    Args:
+        exploration_id: The exploration id given.
```

Comment:
The id of the exploration whose states should be exported.

What about the other args?